cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1247,6 +1247,12 @@ def _add_head_node(self):
                     "base_os": self.config.image.os,
                     "region": self.stack.region,
                     "shared_storage_type": self.config.head_node.shared_storage_type.lower(),
+                    "default_user_home": self.config.deployment_settings.default_user_home.lower()
+                    if (
+                        self.config.deployment_settings is not None
+                        and self.config.deployment_settings.default_user_home is not None
+                    )
+                    else "shared",
```

Comment:
What about having a constant that defines the default value for this setting using the enumeration and reuse such constant here and in the initialization of deployment settings in `cli/src/pcluster/config/common.py`?
In this way we would have a unique place where the default is define.

Reply:
I don't see any precedent for this in the code, but I should use the Enum here!